README.md

```diff
@@ -51,6 +37,27 @@ import '@webmachinelearning/webnn-polyfill';
 <script src="https://cdn.jsdelivr.net/npm/@webmachinelearning/webnn-polyfill/dist/webnn-polyfill.js"></script>
 ```
 
+### Set backend
+
+* When running your tests in Node.js, recommand using CPU backend for its higher numerical precision.
```

Comment:
How about：

 WebNN Polyfill requires setting backend to enable TensorFlow.js.

* When running in Node.js, ....
* When running in browsers, ...